localstack-core/localstack/services/sqs/models.py

```diff
@@ -244,13 +246,18 @@ class MessageMoveTask:
     cancel_event: threading.Event
 
     def __init__(
-        self, source_arn: str, destination_arn: str, max_number_of_messages_per_second: int = None
+        self,
+        source_arn: str,
+        destination_arn: str,
+        max_number_of_messages_per_second: int = None,
+        context: RequestContext = None,
```

Comment:
The `MessageMoveTask` is an async task that is submitted upon receiving a `StartMessageMoveTask`. 

So let's just pass the `StartMessageMoveTask's` context to the submitted `MessageMoveTask` class